### ISSUE TITLE
Add a flag to print version

### ISSUE THREAD

**Issue by ekamil** [OliveTin/OliveTin#175]
**Is your feature request related to a problem? Please describe.**
I'm using the recommended way of installing - with deb packages. I'd like to be able to check which version is installed already in my ansible script to skip installation when not necessary.

**Describe the solution you'd like**
Adding a flag to the command line, example:
```
ansible:~ $ /usr/local/bin/OliveTin --version
2023.10.12
```
Preferred behaviour would to exit after printing.

**Describe alternatives you've considered**
A file somewhere in a well known place with version number?

**Comment by jamesread** [comment]
Sure thing. Just added this, is that OK?

```
root@host: OliveTin --version
INFO OliveTin is just printing the startup message  commit="nocommit" date="nodate" version="dev"
```

**Comment by jamesread** [comment]
@ekamil , Yeah, I know you probably just wanted the "version" string, but I like to think of the version as including the commit hash and date - which is more reliable. 

On release, the compiler sets the version tag to something more useful - it just shows "dev" in my example because I just did a quick development build, but this feature will be included in the next version of OliveTin (which I'm releasing in about 30 mins!)

**Comment by jamesread** [comment]
@ekamil , check the version I released today - 2023.10.24 - is that working OK for you? OK to close this issue?
